src/components/pages/Categories.js

```diff
@@ -1,7 +1,24 @@
 import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { checkStatus } from '../../redux/categories/categories';
 
-const Categories = () => (
-  <button type="button">Check Status</button>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+
+  const status = useSelector((state) => state.category);
+
+  const handleClick = () => {
+    dispatch(checkStatus);
+    console.log(status);
+  };
+
+  return (
+
+    <div>
+      <button type="button" onClick={handleClick}>Check Status</button>
+      <h2>{status}</h2>
+    </div>
```

Comment:
- [ ] According to the project requirement 👇🏻 

![Screenshot from 2023-01-05 00-56-43](https://user-images.githubusercontent.com/79058364/210665618-5c8b8b9d-79b6-4452-8bea-2d3282d952bf.png)

- Actually your `Check Status` button does not work ,please address this 👍🏻